brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced columns - Shape divider theme color issue in editor.

### Description

- Advanced columns > shape divider color when theme color is set.

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Advanced column and content in it.
- Enable shape divider and add a theme color to shape divider its working fine on front.
- Check it is working fine in the editor or not.

### Checklist:
